### PR TITLE
Mgr 1.0.1

### PR DESCRIPTION
**Pull request by garypolhill**
Sorry bug (and mgr 1.0 was badly configured anyway)

### CODE REVIEW

File: libraries.conf [hunk 1]
```diff
@@ -186,6 +195,15 @@ If you’d like to know more about matrices and how they can be used, you might
     homepage: "http://ccl.northwestern.edu/netlogo/6.0.4/docs/matrix.html"
     downloadURL: "https://raw.githubusercontent.com/NetLogo/NetLogo-Libraries/6.1/extensions/matrix-1.1.1.zip"
   }
+  {
+    name: "Java Virtual Machine toolkit"
+    codeName: "mgr"
+    shortDescription: "Various commands for accessing information about the Java Virtual Machine running your NetLogo model"
+    longDescription: """The Java Virtual Machine is the 'computer' that runs your NetLogo model, and Java provides various commands to access information about such things as how much of various different kinds of memory you are using, how many threads are running, and how much CPU time your use of NetLogo is taking. The commands in this extension allow you to access this information, which can be helpful, especially in 'headless' mode, in determining whether you need to make adjustments to default settings for the Java Virtual Machines when running the model."""
+    version: "1.0"
```
Comment (LaCuneta) [inline libraries.conf]: Should this be 1.0.1?

File: libraries.conf [hunk 2]
```diff
@@ -186,6 +195,15 @@ If you’d like to know more about matrices and how they can be used, you might
...
+    homepage: "https://github.com/garypolhill/netlogo-jvmgr/blob/main/README.md"
+    downloadURL: "https://raw.githubusercontent.com/NetLogo/NetLogo-Libraries/6.1/extensions/mgr-1.0.zip"
```
Comment (LaCuneta) [inline libraries.conf]: Link should be to `mgr-1.0.1.zip`, I think.

**Review by LaCuneta — CHANGES_REQUESTED** [review]
Hey Gary!  Thank you so much for the contribution, this looks like a handy extension.  I tested the zip file out and it worked fine. 

I left a couple notes on the `libraries.conf` file, the version is still at 1.0 there.

Also, I think you may need to rebase on to the latest 6.1 branch.  It seems like the branch you used was very old, from before the bit string extension was merged.  If you fetch and pull to the latest `6.1` and then rebase your mgr extension changes on top of that, I think it will clear the conflicts.

Let me know if you have any questions, and thanks again.

File: libraries.conf (hunk 1 above)
Comment (garypolhill) [reply libraries.conf]: Yes. Done

File: libraries.conf (hunk 2 above)
Comment (garypolhill) [reply libraries.conf]: Yes done.

**Comment by LaCuneta** [comment]
Hey Gary, thanks for getting the `libraries.conf` file updated, those changes look good.  We still have the conflict from the bitstring extension commit being there.  I think it should just be a matter of rebasing your local `mgr-1.0.1` branch against our current `6.1` branch then force pushing the results up to your forked repository to update the PR.  Let me know if you need more details on the steps involved.

**Comment by LaCuneta** [comment]
Closing this in favor of PR #48

**Pull request closed without merging**